specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/examples/SubnetCreateWithDelegation.json

```diff
@@ -0,0 +1,60 @@
+{
+  "parameters": {
+    "subnetName": "subnet1",
+    "virtualNetworkName": "vnetname",
+    "resourceGroupName": "subnet-test",
+    "api-version": "2018-08-01",
+    "subscriptionId": "subId",
+    "subnetParameters": {
+      "properties": {
+        "addressPrefix": "10.0.0.0/16"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/subId/resourceGroups/subnet-test/providers/Microsoft.Network/virtualNetworks/vnetname/subnets/subnet1",
+        "name": "subnet1",
+        "properties": {
+          "addressPrefix": "10.0.0.0/16",
+          "provisioningState": "Succeeded",
+          "delegations": [
+            {
+              "name": "myDelegation",
+              "id": "/subscriptions/subId/resourceGroups/subnet-test/providers/Microsoft.Network/virtualNetworks/vnetname/subnets/subnet1/delegations/myDelegation",
+              "properties": {
+                "provisioningState": "Succeeded",

```

Comment:
According the spec you are returning a "Delegation" instead of an "AvailableDelegation" here. "Delegation"'s properties don't contain `provisioningState` and `actions': https://github.com/Azure/azure-rest-api-specs/pull/3693/files#diff-48ae68150e376ac636b1bc2e92b09b32R868.

Reply:
Maybe the service is actually returning "AvailableDelegation"? Please double check.

Reply:
AvailableDelegation and ServiceDelegation are two different resources on the backend

Reply:
Jianghaolu - I do not see any problem here. Rupali is returning Delegations. Can you please confirm and explain more on what you are saying?